subprocess.hpp

```diff
@@ -959,6 +959,17 @@ class Popen
     if (!defer_process_start_) execute_process();
   }
 
+  template <typename... Args>
+  Popen(std::vector<std::string> cmd_args, Args&& ...args):vargs_(cmd_args)
```

Comment:
Not a big deal, but I would like `cmd_args` to be moved into `vargs_`. Otherwise the change looks good. Thanks for contributing!